### ISSUE TITLE
History of frequent kneeling

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10823]
**Preferred term label:**
History of frequent kneeling

**Synonyms**

**Definition (free text, please give PubMed ID)**
Occupations that require frequent kneeling such as roofing, construction workers, plumbers, electricians, landscapers, gardeners, carpenters, housekeepers

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Prepatellar bursitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001157
- label: Occupational exposure to frequent kneeling
- parents: [Occupational exposure history (HP:0035010)](https://hpo.jax.org/app/browse/term/HP:0035010)
- pmid: PMID:18597397
- def: A history of exposure to frequent squatting/kneeling and/or heavy lifting.
- comment: Occupations that require frequent kneeling such as roofing, construction workers, plumbers, electricians, landscapers, gardeners, carpenters, and housekeepers may increase risk of prepatellar bursitis and knee osteoarthritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10823
- synonyms: History of frequent kneeling(NONE)
